python/activator/logger.py

```diff
@@ -23,6 +23,61 @@
 
 import json
 import logging
+import os
+
+import lsst.daf.butler as daf_butler
+
+try:
+    import lsst.log as lsst_log
+except ModuleNotFoundError:
+    lsst_log = None
+
+
+def _parse_log_levels(spec):
+    """Parse a string description of logging levels.
+
+    Parameters
+    ----------
+    spec : `str`
+        A string consisting of space-separated pairs of logger=level
+        specifications. No attempt is made to validate whether the string
+        conforms to this format. "." refers to the root logger.
+
+    Returns
+    -------
+    levels : `list` [`tuple` [`str`, `str`]]
+        A list of tuples whose first element is a logger (root logger
+        represented by `None`) and whose second element is a logging level.
+    """
+    levels = [tuple(s.split("=", maxsplit=1)) for s in spec.split(None)]
+    return [(k if k != "." else None, v) for k, v in levels]
+
+
+def _set_lsst_logging_levels():
+    """Set up standard logging levels for LSST code.
+
+    This function consistently sets both the Python logger and lsst.log.
+    """
+    # Don't call CliLog.initLog because we need different formatters from what
```

Comment:
If there is demand I'm open to moving that code out of butler and into `utils` and allowing more configuration of things like the long log template.

Reply:
I don't think there's demand, at least not yet. Maybe once we've got a clearer picture from commissioning about what we actually need in the service environment.